.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "souffle-addon"]
 	path = souffle-addon
-	url = git@github.com:plast-lab/souffle-addon.git 
+	url = ../../plast-lab/souffle-addon
```

Comment:
Probably want to revert this

Reply:
Hi, it's part of a recent [PR](https://github.com/nevillegrech/gigahorse-toolchain/pull/110). I cloned it locally and it seemed to work. Do you see anything wrong with it?

Reply:
Thanks, I see why it was added now. It will only work if this repositoy is cloned from github so gitlab/other/local clones will need to adjust the submodule to point it to github.

At first I thought someone was developing using a local copy and mistakenly readjusted the submodule.

As far as you are aware it's good.

Reply:
I see, thanks for the info!